.github/workflows/pr_server.yaml

```diff
@@ -0,0 +1,100 @@
+name: pr_server
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'server/**'
```

Comment:
And this directory (.github/workflows) so they run if you just change the test

Reply:
done